certbot-compatibility-test/certbot_compatibility_test/configurators/apache/common.py

```diff
@@ -56,13 +55,14 @@ def load_config(self):
 
     def _prepare_configurator(self):
         """Prepares the Apache plugin for testing"""
-        for k in constants.CLI_DEFAULTS_DEBIAN.keys():
-            setattr(self.le_config, "apache_" + k, constants.os_constant(k))
+        for k in override_debian.DebianConfigurator.OS_DEFAULTS.keys():
```

Comment:
I actually think we should use `entrypoint.ENTRYPOINT` here to use the correct configurator for the environment the tests are run in. I made this change locally and tests still passed.

Reply:
You are right, I'll make the change. All the tests are pinned to a distribution, and no fingerprinting happens runtime anymore so this isn't a big issue *currently* , but might be one in the future.

Reply:
Yeah. I actually didn't realize the code had changed as much as it had, but when I initially wrote the compatibility test, the plan was we'd run it on different distros with different setups which would work by providing the tests metadata about the configs.

Unfortunately, we're not currently making use of that, but maybe we should someday!